src/hooks/useGenerateResult.ts

```diff
@@ -16,15 +16,13 @@ export const useGenerateResult = () => {
       body: JSON.stringify(body),
     })
 
-    if (!response.ok) {
+    if (!response.ok)
```

Comment:
hi @2214962083 
Could you help to update the eslint rules to let the brackets after if statement a required one?

Reply:
sure, i have push the latest code to fix it